studio/settings.py

```diff
@@ -38,6 +38,9 @@
 # Since this file is only used for development, we can have this set to all hosts.
 ALLOWED_HOSTS = ["*"]
 
+# Timezone settings
+USE_TZ = True
+TIME_ZONE = "Europe/Stockholm"
```

Comment:
Line 295 also contains the TIME_ZONE setting and should be removed.

```suggestion
Please remove line 295 with the TIME_ZONE setting.
```